projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/KubernetesService.java

```diff
@@ -2582,6 +2583,7 @@ private Optional<JobDeploymentStatus> mapV1CronJobToDeploymentStatus(
       if (annotations != null) {
         deployment.setLastDeployedBy(annotations.get(JobAnnotation.DEPLOYED_BY.getValue()));
         deployment.setLastDeployedDate(annotations.get(JobAnnotation.DEPLOYED_DATE.getValue()));
+        deployment.setPythonVersion(annotations.get(JobAnnotation.PYTHON_VERSION.getValue()));
```

Comment:
What's the value expected range - is it optional or required?

Reply:
The expected range is the supportedPythonVersions. The python version will either be provided by the user, or the defaultPythonVersion set by the service deployment administrators will be used.